src/config/dynamic-theme-fixes.config

```diff
@@ -7201,6 +7201,19 @@ CSS
 
 ================================
 
+snapeda.com
+
+INVERT
+img[title="SnapEDA"]
+img.part-organization
+
+CSS
+canvas:not(#firstfootprint) {
```

Comment:
I think it's more appropriate to INVERT this.

Reply:
I agree and it has been updated now